config/routes.rb

```diff
@@ -446,6 +447,8 @@
   # in most production settings this is bypassed
   get "letter_avatar_proxy/:version/letter/:letter/:color/:size.png" => "user_avatars#show_proxy_letter"
 
+  get "svg-sprite/:hostname/:version.svg" => "svg_sprite#show", format: false, constraints: { hostname: /[\w\.-]+/ }
```

Comment:
Is there any constraint on the version name possible? I always prefer it if we can.